### PR TITLE
model deployement

### PR DESCRIPTION
**Pull request by riteshkarvaloc**

### CODE REVIEW

**Comment by riteshkarvaloc** [comment]
Backward compatibility for model deployment.

File: dkube/sdk/api.py [hunk 1]
```diff
@@ -1725,19 +1722,11 @@ def create_model_deployment(self, user, name, model=None, modelcatalog=None, ver
         assert stage_or_deploy in [
             "stage", "deploy"], "Invalid value for stage_or_deploy parameter."
 
-        # Fetch the model from modelcatalog
-        if model:
-          mcitem = self.get_modelcatalog_item(user, model=model, version=version)
-        elif modelcatalog:
-          mcitem = self.get_modelcatalog_item(user, modelcatalog=modelcatalog, version=version)         
+        # Fetch the modelrepo from modelcatalog
+        catalog_model = None
```
Comment (songole) [inline dkube/sdk/api.py:1726]: is this variable needed?

Comment (riteshkarvaloc) [reply dkube/sdk/api.py]: @songole it was not required. I removed it now.

File: dkube/sdk/api.py [hunk 2]
```diff
@@ -1695,10 +1695,7 @@ def create_model_deployment(self, user, name, model=None, modelcatalog=None, ver
                     User readable description of the deployment
 
                 model
-                    Name of the model to be deployed
-                    
-                modelcatalog
-                     model catalog name
+                    Name of the model to be deployed, Model need to be published
```
Comment (deepio-oc) [inline dkube/sdk/api.py:1699]: this help text sounds confusing to me. We could leave as it was earlier or clarify this better.

Comment (riteshkarvaloc) [reply dkube/sdk/api.py:1699]: @deepio-oc The help test is reverted.

File: dkube/sdk/api.py [hunk 3]
```diff
@@ -1725,19 +1722,10 @@ def create_model_deployment(self, user, name, model=None, modelcatalog=None, ver
         assert stage_or_deploy in [
             "stage", "deploy"], "Invalid value for stage_or_deploy parameter."
 
-        # Fetch the model from modelcatalog
-        if model:
-          mcitem = self.get_modelcatalog_item(user, model=model, version=version)
-        elif modelcatalog:
-          mcitem = self.get_modelcatalog_item(user, modelcatalog=modelcatalog, version=version)         
+        # Fetch the modelrepo from modelcatalog
```
Comment (songole) [inline dkube/sdk/api.py:1725]: is this comment right?

Comment (riteshkarvaloc) [reply dkube/sdk/api.py]: @songole it's removed.

**Review by deepio-oc — APPROVED** [review]

**Pull request merged**